tests/integration_tests/test_mlflow.py

```diff
@@ -398,8 +398,10 @@ def _objective_func(trial: optuna.trial.Trial) -> float:
         mlflow.log_metric(metric_name, metric)
         return (x - 2) ** 2 + (y - 25) ** 2 + z
 
+    tracked_objective = mlflc.track_in_mlflow()(_objective_func)
```

Comment:
Please tell me why we have to stop using decorator syntax and manually invoke wrapping function?

Reply:
To access the docstring and function name of `objective`'s `name` and `docstring`.

Reply:
Note that decorator syntax and wrapping the function are equivalent.